addons/stock/migrations/11.0.1.1/pre-migration.py

```diff
@@ -37,10 +37,21 @@ def delete_quants_for_consumable(env):
     )
 
 
+def drop_slow_constraint(env):
+    """Removing this constraint, that doesn't affect new data structure, we
+    get tons of more performance on quant removal.
```

Comment:
Please add to the comment that the reason this does not affect the data model is that it belongs to an obsolete model.

Reply:
Changed. You can squashed on merge for removing the extra commit as I edited by UI.